### PR TITLE
Random spontaneous cleanup PR

### PR DESCRIPTION
**Pull request by PJB3005**
Just fixing a ton of warnings and other misc cleanup.

### CODE REVIEW

File: Content.Client/Administration/Managers/ClientAdminManager.cs [hunk 1]
```diff
@@ -15,11 +15,14 @@ public sealed class ClientAdminManager : IClientAdminManager, IClientConGroupImp
         [Dependency] private readonly IClientNetManager _netMgr = default!;
         [Dependency] private readonly IClientConGroupController _conGroup = default!;
         [Dependency] private readonly IResourceManager _res = default!;
+        [Dependency] private readonly ILogManager _logManager = default!;
+
 
         private AdminData? _adminData;
```
Comment (0x6273) [inline Content.Client/Administration/Managers/ClientAdminManager.cs:21]:
```suggestion
        [Dependency] private readonly ILogManager _logManager = default!;

        private AdminData? _adminData;
```

**Comment by potato1234x** [comment]
place your bets on how many things this will break now

**Comment by EmoGarbage404** [comment]
> place your bets on how many things this will break now :godo:

if you're just going to be annoying and unhelpful do it somewhere that isn't on the development github.

**Comment by lzk228** [comment]
names that we deserve

**Comment by TemporalOroboros** [comment]
What's the warning count after this PR? Master is currently at 1795 for all of content.

**Comment by PJB3005** [comment]
2457 on master -> 2142 with this PR, but that's including engine.

**Pull request merged**